accounting/conversions.go

```diff
@@ -66,14 +64,7 @@ func getConversion(ctx context.Context, startTime, endTime time.Time,
 
 	// Create a wrapper function which can be used to get individual price
 	// points from our set of price data as we create our report.
-	return func(amtMsat int64, ts time.Time) (decimal.Decimal, error) {
-		price, err := fiat.GetPrice(prices, ts)
-		if err != nil {
-			return decimal.Zero, err
-		}
-
-		return fiat.MsatToUSD(
-			price.Price, lnwire.MilliSatoshi(amtMsat),
-		), nil
+	return func(ts time.Time) (*fiat.USDPrice, error) {
+		return fiat.GetPrice(prices, ts)
```

Comment:
Looks like something can be squashed if this function is rewritten again in the pr

Reply:
I understand not wanting to touch code twice in a PR, but sometimes it feels like the cost of the squash (in this case, squashing together changes to completely different packages, making potential future bug hunting more difficult) is more than the benefit of not touching 5 lines twice.